### PR TITLE
Added config file for Welcome BOT

### PR DESCRIPTION
**Pull request by muthuannamalai12**
I have added a welcome bot config file having a proper message that will show up when any user will open up an issue or pull-request for the first time.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
```
Comment (praveenscience) [inline .github/config.yml:7]: Let's have a space before the emojis, please?

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  
+
```
Comment (praveenscience) [inline .github/config.yml:9]: Do we need two line spaces or one? If one, can we just remove the extra line?

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉 for opening your very first issue in this project. Our project aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
+  Please adhere to our [Code of Conduct](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/Code_of_Conduct.md).
+  Please make sure not to start working on the issue, unless you get assigned to it.😄
+  
+
```
Comment (praveenscience) [inline .github/config.yml:14]: Do we need two line spaces or one? If one, can we just remove the extra line?

File: .github/config.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  Please make sure not to start working on the issue, unless you get assigned to it.😄
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+
```
Comment (praveenscience) [inline .github/config.yml:21]: Do we need two line spaces or one? If one, can we just remove the extra line?

File: .github/config.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉 for opening your very first issue in this project. Our project aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
```
Comment (praveenscience) [inline .github/config.yml:10]: Let's have a space before the emojis, please?

File: .github/config.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  Please adhere to our [Code of Conduct](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/Code_of_Conduct.md).
+  Please make sure not to start working on the issue, unless you get assigned to it.😄
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
```
Comment (praveenscience) [inline .github/config.yml:19]: Let's have a space before the emojis, please?

File: .github/config.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉 for opening your very first issue in this project. Our project aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
+  Please adhere to our [Code of Conduct](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/Code_of_Conduct.md).
+  Please make sure not to start working on the issue, unless you get assigned to it.😄
```
Comment (praveenscience) [inline .github/config.yml:12]: Let's have a space before the emojis, please?

File: .github/config.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉 for opening your first pull request. Our project aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
+  Please adhere to our [Code of Conduct](https://github.com/praveenscience/Internship-LMS-FrontEnd/blob/main/Code_of_Conduct.md).🙌 We will get back to you as soon as we can 😄.
```
Comment (praveenscience) [inline .github/config.yml:23]: Let's have a space before the emojis, please?

File: .github/config.yml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+  
+
```
Comment (praveenscience) [inline .github/config.yml:25]: Do we need two line spaces or one? If one, can we just remove the extra line?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Thanks. Looks almost nice.

**Review by praveenscience — APPROVED** [review]
LGTM! 👍

**Review by Salonii02 — APPROVED** [review]

**Pull request merged**